oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/TimeBucket.java

```diff
@@ -17,14 +17,18 @@
 
 package org.apache.skywalking.oap.server.core.analysis;
 
-import java.util.Calendar;
 import org.apache.skywalking.oap.server.core.UnexpectedException;
 
 /**
  * @author peng-yongsheng
  */
 public class TimeBucket {
 
+    public static final int SECOND_BUCKET = 60;
+    public static final int HOUR_BUCKET = 60 * 60;
+    public static final int DAY_BUCKET = 60 * 60 * 24;
+    public static final int MONTH_BUCKET = 60 * 60 * 24 * 30;
```

Comment:
Not every month is 30 days.

Reply:
Agree, @arugal got the point, this calculation is not accurate.

Reply:
It ’s really a bit problematic for the month, but it ’s not currently used, but the day, hour, and minute should be fine.

Reply:
Month is used for all metrics in default. Also, if you change this, all query(s) are affected. Check the timebucket usage in the query stage.

Reply:
Ok, let me take a look